tests/foreman/cli/test_host.py

```diff
@@ -3321,3 +3321,23 @@ def test_positive_errata_list_of_sat_server(self):
         hostname = ssh.command('hostname').stdout[0]
         host = Host.info({'name': hostname})
         self.assertIsInstance(Host.errata_list({'host-id': host['id']}), list)
+
+
+class EncDumpTestCase(CLITestCase):
+    """Tests for Dump host's ENC YAML"""
+
+    @tier1
+    def test_positive_dump_enc_yaml(self):
+        """Dump host's ENC YAML. Check BZ for details.
+
+        :id: 50bf2530-788c-4710-a382-d034d73d5d4d
+
+        :expectedresults: Dump host's ENC YAML
+
+        :BZ: 1372731
+
+        :CaseImportance: Critical
+        """
```

Comment:
we need an other field in doc string 
```
:customerscenario: true
```
after merging do not to set bug flag qe_test_coverage to +